lib/importproject.h

```diff
@@ -81,6 +81,7 @@ class CPPCHECKLIB ImportProject {
         void setIncludePaths(const std::string &basepath, const std::list<std::string> &in, std::map<std::string, std::string, cppcheck::stricmp> &variables);
     };
     std::list<FileSettings> fileSettings;
+    Type mType;
```

Comment:
I think that Travis fails because you do not initialize this member variable properly everywhere.

Since it's a public member variable I think that the name shouldnt start with "m".

Reply:
Mmh, it confusing because the whole class is mentioned as "uninitialized".
I've created constructor for `ImportProject`.